Submitted By: Jonathan Norman (jonathan at bluesquarelinux dot co dot uk)
Date: 2012-02-17
Initial Package Version: 2.30
Origin: Myself
Upstream Status: None
Description: Updated filenames and online locations of port numbers and protocol files.

diff -Naur iana-etc-2.30.orig/Makefile iana-etc-2.30/Makefile
--- iana-etc-2.30.orig/Makefile	2012-02-17 22:57:44.000000000 +0000
+++ iana-etc-2.30/Makefile	2012-02-17 23:24:00.000000000 +0000
@@ -42,12 +42,12 @@
 	    protocol-numbers.iana port-numbers.iana
 
 protocol-numbers.iana:
-	$(AWK) -f get.gawk -v file=protocol-numbers >protocol-numbers.iana
-	rm -f protocol-numbers
+	$(AWK) -f get.gawk -v file=protocol-numbers/protocol-numbers.txt >protocol-numbers.iana
+	rm -f protocol-numbers.txt
 
 port-numbers.iana:
-	$(AWK) -f get.gawk -v file=port-numbers >port-numbers.iana
-	rm -f port-numbers
+	$(AWK) -f get.gawk -v file=service-names-port-numbers/service-names-port-numbers.txt >port-numbers.iana
+	rm -f service-names-port-numbers.txt
 
 protocol-numbers:
 ifeq (protocol-numbers.iana, $(wildcard protocol-numbers.iana))
